UIDokan/Sources/UIDokan/Style/UIButtonStyle.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  File.swift
+//  
+//
+//  Created by marwa on 13/07/2022.
+//
+
+import Foundation
+import UIKit
+
+
+extension UIButton {
+    
+    public enum buttonStyle {
+        case filledBlue
+        case filledGray
+        case filledRed
+        case border
+        case plainBlueProductTitle
+        case plainBlueSmallBtn
+        case plainBlack
+    }
+    
+    public func applyButtonStyle(_ style:buttonStyle, buttonName: String) {
```

Comment:
Separation of Concerns Principle (SOC) - Should should be allowed to update style only. 
```suggestion
    public func applyButtonStyle(_ style:buttonStyle) {
```